models/review.model.js

```diff
@@ -1,6 +1,12 @@
 const mongoose = require('mongoose');
 
 var reviewSchema = new mongoose.Schema({
+    reviewer:{
```

Comment:
can you also change the post review api in user.js. [https://github.com/zxtariel0512/Second-Hand-Transaction-Webapp-for-NYU-Students/blob/0c8ca345ca983b424d9388acc450cd341c2e6138/routes/user.js#L112](url)
add one more attribute in review object which is reviewer. and its value is req.params.reviewer